test/utils/constants.test.js

```diff
@@ -1,5 +1,19 @@
-/* eslint-env node, mocha */
-/* eslint no-console: 0*/
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
```

Comment:
Adding the copyright might be better in a dedicated repo-encompassing PR where it's added/modified to all related files instead of a code-change PR, but if it's gonna be here, we might as well make it 2017, right?